packages/ui-select/src/Select/index.tsx

```diff
@@ -673,6 +673,13 @@ class Select extends Component<SelectProps> {
     const highlightedOptionId = this.highlightedOptionId
     const selectedOptionId = this.selectedOptionId
 
+    // props.width should apply to the whole component, not just the input in it
+    const extendedStyle = {
+      ...(styles?.select as object),
+      width: this.props.width || '100%',
+      display: 'inline-block'
+    }
```

Comment:
CSS should be in `styles.ts`. This should be in the `select` object there.

Reply:
good point, moved it there